### PR TITLE
create ci-cd yaml files and add global variables in R script to remove warnings 

### PR DESCRIPTION
**Pull request by Kingslin0810**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -8,7 +8,7 @@ Authors@R: c(
            email = "allyson.m.stoll@gmail.com"),
     person(given = "Kingslin",
            family = "Lv",
-           role = c("aut", "cre"),
+           role = c("aut"),
```
Comment (jo4356) [inline DESCRIPTION]: Removing "cre" will cause `check()` to not pass. We need to have at least one person with "cre" role.

**Review by jo4356 — CHANGES_REQUESTED** [review]

File: R/globals.R [hunk 2]
```diff
@@ -0,0 +1,3 @@
+utils::globalVariables(c("Freq", "AtomicMass", "Freq.x",
```
Comment (jo4356) [inline R/globals.R:1]: I don't think we should declare these as global variables. Some of these are column names used in dataframes.

File: DESCRIPTION (hunk 1 above)
Comment (Kingslin0810) [reply DESCRIPTION]: I was removing as I got the ci-cd action running error; let me add it back to see if there is another error in the description file.

**Review by Kingslin0810 — COMMENTED** [review]
add "cre" maintainer role back to description

**Review by jo4356 — CHANGES_REQUESTED** [review]

File: R/globals.R (hunk 2 above)
Comment (Kingslin0810) [reply R/globals.R:1]: I was getting this warning which prevents ci-cd github action and so those columns names used in dataframes are added as the global variables. https://community.rstudio.com/t/how-to-solve-no-visible-binding-for-global-variable-note/28887

**Pull request merged**